.vscode/tasks.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
.vscode folder is automatically added during the project build. We don't need to have this in repo. It's better to add this folder to .gitignore